models/iaf_psc_exp.h

```diff
@@ -99,6 +96,13 @@ class Network;
    For details, please see IAF_Neruons_Singularity.ipynb in the
    NEST source code (docs/model_details).
 
+   If the current event corresponding to the piecewise constant
+   external current is received at port 1 the current is filtered by
+   the synapse. Hereby the time constant of the post-synaptic
+   excitatory currents is used. This filtering can be used for a
+   direct implementation of the diffusion approximation, i.e.,
+   replacing incoming spike trains with white noise.
```

Comment:
@jschuecker Thanks for the more detailed explanation. I would be yet more explicit, e.g.

"iaf_psc_exp can handle current input in two ways: Current input through receptor_type 0 are handled as stepwise constant current input as in other iaf models, i.e., this current directly enters the membrane potential equation. Current input through receptor_type 1, in contrast, is filtered through an exponential kernel with the time constant of the excitatory synapse, tau_syn_ex. For an example application, see [4; reference to paper inserted below]."

I am a bit concerned about the statement that this provides a "direct implementation ... with white noise". Since the noise generator generates stepwise constant input, the input is never really white noise, see doc/model_details/noise_generator.ipynb.